app/app/src/main/res/values/styles.xml

```diff
@@ -1,18 +1,29 @@
 <resources>
+
     <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
     </style>
+
     <style name="actionButtonStyle">
         <item name="android:layout_height">wrap_content</item>
         <item name="android:layout_width">@dimen/activity_button_width</item>
         <item name="android:backgroundTint">@color/action_button_color</item>
+        <item name="android:textSize">20dp</item>
```

Comment:
1-. all TextSizes will be sp instead of dp
2-. do not hardcode pls, use dimens

Reply:
done, thanks